.travis.yml

```diff
@@ -9,7 +9,8 @@ env:
 - ARCH=i686
```

Comment:
Is 32 bit required?

Reply:
mc is not decided yet @balamurugana as a client perhaps its okay so we kept it.

Reply:
Actually you are right we don't need.. removing.